lighthouse-core/runner.js

```diff
@@ -375,6 +380,10 @@ class Runner {
       'metrics', // the sub folder that contains metric names
       'metrics/lantern-metric.js', // lantern metric base class
       'metrics/metric.js', // computed metric base class
+
+      // Computed artifacts switching to the new system.
+      'new-computed-artifact.js',
+      'manifest-values.js',
```

Comment:
This fn is for the gulpfile. wont browserify already discover this manifest-values module without being added manually?

Reply:
the diff obscures it, but this is the list of filenamesToSkip :)